app/assets/stylesheets/sign_in.scss

```diff
@@ -10,10 +10,10 @@ body#sign-in {
   margin-top: 7rem;
 
   h1 {
-    color: #f5f5f5;
-    letter-spacing: -1.5rem;
-    line-height: 16rem;
-    font-size: 17rem;
+    color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`